### PR TITLE
Add --enable-microsoft to configure, enable native installation kits

### PR DESCRIPTION
**Pull request by jeffaco**
@Microsoft/ostc-devs @Microsoft/omi-devs 

First stab at OMI building it's own packages. This is tested on Linux and appears to work fine, running a pbuild now for platform breadth.

Note that this shelveset is designed to work in concert with SCXcore changes as well, review that as well please.

NOTE: After this, I'll make other (unrelated) changes, like `./configure --dev` actually configuring the way that our standard build will, so the appropriate tests are run by regress. But that's separate to OMI building it's own packages, so I wanted to make that a separate commit.

### CODE REVIEW

File: Unix/configure [hunk 1]
```diff
@@ -361,6 +395,89 @@ buildtool="$root/buildtool"
...
+    esac
+
+    openssl098_cflags=`PKG_CONFIG_PATH=$openssl098_libdir/pkgconfig $pkgconfig --cflags openssl`
+    openssl098_libs=`PKG_CONFIG_PATH=$openssl098_libdir/pkgconfig $pkgconfig --libs openssl`
+    openssl100_cflags=`PKG_CONFIG_PATH=$openssl100_libdir/pkgconfig $pkgconfig --cflags openssl`
+    openssl100_libs=`PKG_CONFIG_PATH=$openssl100_libdir/pkgconfig $pkgconfig --libs openssl`
+
+    echo "checking for local installation of openssl version 0.9.8 on system ..."
+    if [ ! -d "$openssl098_dir" ]; then
+	echo "did not find ${openssl098_dir}, which is required."
```
Comment (paulcallen) [inline Unix/configure]: use spaces instead of tabs

File: Unix/configure [hunk 2]
```diff
@@ -2231,20 +2330,41 @@ echo "created $fn"
...
-export DATADIR=$datadir
 export ROOT=$root
+export ENABLE_ULINUX=$enable_ulinux
+
+OMI_CONFIGURE_QUALS=--enable-microsoft --disable-makefile-gen
 
 all:
+ifeq (\$(ENABLE_ULINUX),1)
+
+	@echo '========================= Performing Building OMI (0.9.8)'
```
Comment (paulcallen) [inline Unix/configure]: can you say ssl version 0.9.8

File: Unix/configure (hunk 1 above)
Comment (jeffaco) [inline Unix/configure]: Changed to spaces rather than tabs.

File: Unix/configure (hunk 2 above)
Comment (jeffaco) [inline Unix/configure]: Okay, changed to `Performing Building OMI (SSL 0.9.8)`. Similar change for SSL 1.0.0.

**Pull request merged**